client/scss/base/_icons.scss

```diff
@@ -14,6 +14,14 @@
   stroke: color('black');
 }
 
+.icon--fill-negative {
```

Comment:
'negative' doesn't make me automatically think of 'white'. Would `.icon--fill-white` be clearer?